src/Statement/StatementGuid.php

```diff
@@ -55,6 +61,13 @@ public function getEntityId() {
 		return $this->entityId;
 	}
 
+	/**
+	 * @return string
+	 */
+	public function getGuid() {
```

Comment:
We probably also want to call this getGuidPart.

Otherwise someone will get confused at some point as we have $statement->getGuid which returns a string which is the whole guid, and then $guid->getGuid which is just the later part.